### PR TITLE
Homepage design

### PR DESCRIPTION
**Pull request by srivig**

### CODE REVIEW

File: mod/wikirate/lib/javascript/test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            console.log (cnt + "Liked");
+        }
+    }
+    else
+    {	console.log('Total HREF Links Processed: ' + anchorTag.length);
+        clearInterval(TimerFunCall);
+    }
+    i=i+1;
+    return i;
+}
```
Comment (xithan) [inline mod/wikirate/lib/javascript/test.js]: what is this js file about?

Comment (srivig) [reply mod/wikirate/lib/javascript/test.js]: I have no idea. I am deleting it

Comment (xithan) [reply mod/wikirate/lib/javascript/test.js]: 😄

Comment (srivig) [reply mod/wikirate/lib/javascript/test.js]: ahh, I just remember, I wrote that script to 'like' all the birthday wishes on my facebook wall. Looks like I saved it in WikiRate dev directory >.<

**Pull request merged**